### ISSUE TITLE
Support for more timeframes on GET /trade/bucketed

### ISSUE THREAD

**Issue by yazeed** [BitMEX/api-connectors#223]
Please support more timeframes on GET /trade/bucketed, the 3m, 10m, and 15m, would be great. They're needed by many trading strategies.

**Comment by yazeed** [comment]
I have, already. Thanks for your response,

Friday, September 28, 2018, 12:26 AM +0300 from Set Kyar Wa Lar  <notifications@github.com>:

**Comment by redallica** [comment]
Hey @yazeed 

Did you get any feedback from Bitmex support on this matter ?
Thanks

**Comment by yazeed** [comment]
@redallica, yes the bitmex support agent just told me that he'd forward my feedback to the technical department responsible for such a decision. I basically requested all missing timeframes to give us better control in strategy building, specially when it comes to timeframe correlation

**Comment by yazeed** [comment]
@redallica on the same note, I had to have my trading bot go about rebuilding 3h candles manually using 1h candles.
